x-pack/legacy/plugins/watcher/public/np_ready/application/models/action/webhook_action.js

```diff
@@ -11,23 +11,26 @@ import { i18n } from '@kbn/i18n';
 export class WebhookAction extends BaseAction {
   constructor(props = {}) {
     super(props);
-
     const defaultJson = JSON.stringify(
       { message: 'Watch [{{ctx.metadata.name}}] has exceeded the threshold' },
       null,
       2
     );
     this.body = get(props, 'body', props.ignoreDefaults ? null : defaultJson);
-
     this.method = get(props, 'method');
     this.host = get(props, 'host');
     this.port = get(props, 'port');
+    this.scheme = get(props, 'scheme');
     this.path = get(props, 'path');
     this.username = get(props, 'username');
     this.password = get(props, 'password');
     this.contentType = get(props, 'contentType');
 
-    this.fullPath = `${this.host}:${this.port}${this.path}`;
+    if (typeof this.path !== 'undefined') {
```

Comment:
Fwiw, this is a very narrow check for the value of `undefined` (so values like `null` or `0` will pass the `if` check) which should be fine in this case but something like 

```ts
this.fullPath = `${this.host}:${this.port}${this.path ? '/' + this.path : ''}`;
```

would be more robust because it's a default for all falsey values. I don't expect `this.path` to be anything other than `undefined` or a `String` in this case.

Reply:
@jloleysens thanks for the advice, I've updated my code and uploaded it